setup.py

```diff
@@ -47,6 +44,7 @@
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
```

Comment:
Is this kosher? We're technically under the Dropbox License which isn't far off from I think the Apache license.